modules/nodes-pods-vertical-autoscaler-tuning.adoc

```diff
@@ -0,0 +1,164 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-vertical-autoscaler.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="nodes-pods-vertical-autoscaler-tuning_{context}"]
+= Performance tuning the VPA Operator
+
+As a cluster administrator, you can tune the performance of your Vertical Pod Autoscaler Operator (VPA) to control the rate at which the VPA makes requests of the Kubernetes API server and to specify the CPU and memory resources for the VPA recommender, updater, and admission controller component pods. 
+
+Additionally, you can configure the VPA Operator to monitor only those workloads that are being managed by a VPA CR. By default, the VPA Operator monitors every workload in the cluster. This allows the VPA Operator accrue historical data for workloads, which the Operator then uses when a new VPA CR is created for a workload. However, this causes the VPA Operator to use significant CPU and memory, which could cause the Operator to fail. By configuring the Operator to monitor only workloads with a VPA CR, you can save on CPU and memory resources. One trade-off is that if you have a workload that has been running, and you create a VPA CR to manage it, the VPA Operator does not have any historical data for that workload, so the recommendations are not as useful at first. 
+
+These settings allow you to make sure the VPA has sufficient resources to operate at peak efficiency and to prevent throttling and a delay in pod admissions, particulalry on larger clusters. 
+
+You can configure these settings for the VPA components by editing the `VerticalPodAutoscalerController` custom resource (CR):
+
+* To prevent throttling and pod admission delays, specify the queries-per-second and burst rates for VPA requests of the Kubernetes API server by using the 'kube-api-qps' and `kube-api-burst` parameters.
+
+* To ensure sufficient CPU and memory, specify CPU and memory requests for VPA component pods by using the standard `cpu` and `memory` resource requests.
+
+* To configure the VPA Operator to monitor only those workloads that are being managed by a VPA CR, set the `memory-saver` parameter to `true` for the recommender.    
+
+The following example VPA controller CR sets the VPA API QPS and burts rates, configures the component pod resource requests, and sets `memory-saver` to `true` for the recommender: 
+ 
+.Example `VerticalPodAutoscalerController` CR
+[source,yaml]
+----
+apiVersion: autoscaling.openshift.io/v1
+kind: VerticalPodAutoscalerController
+metadata:
+  name: default
+  namespace: openshift-vertical-pod-autoscaler
+  resourceVersion: '61871'
+  uid: 9c68a2c9-843f-42bc-8030-b357f70e06f7
```

Comment:
I think it is probably best to leave resourceVersion and uid off of the example.